### PR TITLE
Event pages for upcoming bytesize talks

### PR DESCRIPTION
**Pull request by ewels**

### CODE REVIEW

**Review by mashehu — APPROVED** [review]

File: markdown/events/2022/bytesize-40-software-packaging.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: "Bytesize 40: Software packaging"
+subtitle: Alexander Peltzer - Boehringer Ingelheim Pharma GmbH & Co. KG, Germany
+type: talk
+start_date: "2022-04-05"
+start_time: "13:00 CET"
```
Comment (mashehu) [inline markdown/events/2022/bytesize-40-software-packaging.md:6]:
```suggestion
start_time: "13:00 CEST"
```

File: markdown/events/2022/bytesize-40-software-packaging.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+end_date: "2022-03-01"
+end_time: "13:30 CET"
```
Comment (mashehu) [inline markdown/events/2022/bytesize-40-software-packaging.md:8]:
```suggestion
end_time: "13:30 CEST"
```

**Review by mashehu — APPROVED** [review]

File: markdown/events/2022/bytesize-36-multiqc.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+title: "Bytesize 36: Customising your MultiQC reports"
+subtitle: Phil Ewels - National Genomics Infrastructure / SciLifeLab, Sweden
+type: talk
+start_date: "2022-03-08"
+start_time: "13:00 CET"
+end_date: "2022-03-01"
```
Comment (mashehu) [inline markdown/events/2022/bytesize-36-multiqc.md:7]:
```suggestion
end_date: "2022-03-08"
```

File: markdown/events/2022/bytesize-37-gathertown.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+---
+title: "Bytesize 37: Gather Town"
+subtitle: James A. Fellows Yates - LMU Munich / MPI-EVA
+type: talk
+start_date: "2022-03-15"
+start_time: "13:00 CET"
+end_date: "2022-03-01"
```
Comment (mashehu) [inline markdown/events/2022/bytesize-37-gathertown.md:7]:
```suggestion
end_date: "2022-03-15"
```

File: markdown/events/2022/bytesize-40-software-packaging.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+---
+title: "Bytesize 40: Software packaging"
+subtitle: Alexander Peltzer - Boehringer Ingelheim Pharma GmbH & Co. KG, Germany
+type: talk
+start_date: "2022-04-05"
+start_time: "13:00 CET"
+end_date: "2022-03-01"
```
Comment (mashehu) [inline markdown/events/2022/bytesize-40-software-packaging.md:7]:
```suggestion
end_date: "2022-04-05"
```

**Comment by ewels** [comment]
Oops, maybe merged that a little early!

**Comment by mashehu** [comment]
Yeah, sorry, I hit approve a little to early 😀

File: markdown/events/2022/bytesize-40-software-packaging.md (hunk 5 above)
Comment (ewels) [reply markdown/events/2022/bytesize-40-software-packaging.md:7]: I have a feeling that the `end_date` can actually just be omitted if it's a single day event..? But my memory is a bit rusty and I guess it doesn't hurt..

**Comment by ewels** [comment]
Suggestions applied in https://github.com/nf-core/nf-co.re/pull/1050

**Pull request merged**